diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+exercise, Jason, JasonWeeks
```

Comment:
1 of 3 correct

`teamMember['name']` returns an object that has keys and values : `{ given: 'Jason', surname: 'Weeks'};` is the correct value returned

`teamMember['name']['surname']` looks through the object `name` and finds a key of `surname:` which has a value of 'Weeks'